packages/ove-service-layout/src/routes/render.js

```diff
@@ -0,0 +1,22 @@
+let {convertError} = require("../util");
+
+let {validateRequest} = require("../validator/validator");
+let {layoutManager} = require("../layout/manager");
+
+let logger = require("debug")("layout:render");
+
+let {oveClient} = require("../routes/ove");
+
+exports.renderRoute = (req, res) => {
+    validateRequest("render", req.body);
+
+    logger("Request", req.body);
+
+    oveClient.getConfiguration(req.body["ove-space"]).then(config => {
+        res.status(200).json(layoutManager.renderCanvas(req.body, config.geometry));
```

Comment:
Use HTTP Constants instead of 200, 500 etc. That makes code slightly more readable and also consistent across the board.